<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Author_Sorting</class>
 <widget class="QWidget" name="Author_Sorting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>491</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonReturn">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBegin">
        <property name="text">
         <string>开始查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>         发文章数量前100作者</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>7</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>8</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>9</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>10</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>11</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>12</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>13</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>14</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>15</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>16</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>17</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>18</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>19</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>20</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>21</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>22</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>23</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>24</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>25</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>26</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>27</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>28</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>29</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>30</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>31</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>32</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>33</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>34</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>35</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>36</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>37</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>38</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>39</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>40</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>41</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>42</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>43</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>44</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>45</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>46</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>47</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>48</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>49</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>50</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>51</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>52</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>53</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>54</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>55</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>56</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>57</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>58</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>59</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>60</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>61</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>62</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>63</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>64</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>65</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>66</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>67</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>68</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>69</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>70</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>71</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>72</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>73</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>74</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>75</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>76</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>77</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>78</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>79</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>80</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>81</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>82</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>83</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>84</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>85</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>86</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>87</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>88</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>89</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>90</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>91</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>92</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>93</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>94</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>95</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>96</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>97</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>98</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>100</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>作者名</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>发表文章数</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
